uiowa/uiowa01
# Feature: Clean up related to profiles

This update removes the previous version of the collegiate profile, which was based on Lightning and no longer in use; renamed the collegiate_basic profile to collegiate; and moves it into a separate project called [uiowa/collegiate_profile](https://github.com/uiowa/collegiate_profile) and makes it an external dependency.

Please note there is additional work that needs to be done to make the profile truly independent of this project, as it still depends on modules that are part of this project.

# How to test
`drush @theming.local si collegiate`
Spot check functionality to make sure nothing is broken - I'm not too worried that this would affect any existing functionality because I did a project-wide find and replace for "collegiate_basic" and there were very few instances that needed to be changed.
